src/CSnakes.Runtime/PythonRuntimeException.cs

```diff
@@ -47,6 +48,14 @@ public string[] PythonStackTrace
 
     private static string[] FormatPythonStackTrace(PyObject pythonStackTrace)
     {
+        if (!CPythonAPI.IsInitialized)
+        {
+            // For why check and bail out here, see:

```

Comment:
Remove the comment, this pattern is used in lots of places.

Reply:
Removed with d2df3543b372da280aff02d3ca98bb8b445cd0fe.